src/client/AccountsClient.js

```diff
@@ -48,13 +53,75 @@ export class AccountsClient {
       middleware,
     });
   }
-
   getState(): Map<string, any> {
     return this.store.getState().get('accounts');
   }
   user(): UserObjectType {
     return this.getState().get('user').toJS();
   }
+  tokens(): TokensType {
+    return {
+      accessToken: localStorage.getItem(ACCESS_TOKEN),
+      refreshToken: localStorage.getItem(REFRESH_TOKEN),
+    };
+  }
+  clearTokens() {
+    localStorage.removeItem(ACCESS_TOKEN);
+    localStorage.removeItem(REFRESH_TOKEN);
+  }
+  clearUser() {
+    this.store.dispatch(clearUser());
+  }
+  async resumeSession(): Promise<void> {
+    const { accessToken } = this.tokens();
+    if (accessToken) {
+      try {
+        const decodedAccessToken = jwtDecode(accessToken);
```

Comment:
Is this the right way to do this?   We're technically validating the login here on the client, correct?   This means with a malicious client we can never expire them since we only check the expiration via the client's token.

This is my first time looking at this code, so definitely could be overlooking something.